test/db/analysis/vars

```diff
@@ -427,11 +427,11 @@ afvR
       arg1  0x6
       arg2  0x8
       arg3  0xc
-   var_10h  0x4,0x2c
+   var_10h
```

Comment:
It's not read anymore? What I wonder, why it's still listed then as the output of this command?

Reply:
@XVilka The reason those "empty" vars are added is due to https://github.com/rizinorg/rizin/commit/e0b93d46ebf3a5fc43e85db7b4c796e2dda4e540
And since then standard behavior (see the tests there).

In `esilbreak_reg_write()`:
https://github.com/rizinorg/rizin/blob/8e29b959b86a35bbbfed599989f077dba6e0ebd5/librz/core/canalysis.c#L3063-L3071

a variable is added. But it never gets checked if the register write is something like `sp + 0`.
See:
https://github.com/rizinorg/rizin/blob/8e29b959b86a35bbbfed599989f077dba6e0ebd5/librz/core/canalysis.c#L2995-L2998

I can fix this, but it would break more tests and IMHO it's out of scope here. Especially if we change the var detection soon anyway.

Reply:
Also add a link to this issue in the test too, like in other places please